client/me/concierge/reschedule/confirmation-step.js

```diff
@@ -24,9 +24,7 @@ class ConfirmationStep extends Component {
 
 		return (
 			<Confirmation
-				description={ translate(
-					'We will send you a calendar invitation and an email with information on how to prepare.'
-				) }
+				description={ translate( 'We will send you an email with information on how to prepare.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'We will send you an email with information on how to get prepared.' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/concierge-reschedule)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).